webapp/sources/ldap-inventory/inventory-provisioning-core/src/main/scala/com/normation/inventory/ldap/provisioning/DefaultReportSaver.scala

```diff
@@ -83,11 +83,15 @@ class DefaultReportSaver(
         t3  <- currentTimeMillis
         _   <- InventoryProcessingLogger.timing.trace(s"Saving node: ${t3 - t2} ms")
 
-        d3  <- ZIO.foreach(report.vms) { x =>con.saveTree(mapper.treeFromMachine(x), deleteRemoved = true) }
+        d3  <- con.save(mapper.processesFromNode(report.node), removeMissingAttributes = false)
```

Comment:
I *think* (not sure) that we could even ignore errors on that one (or at least timeout errors) (with a `catchAll(err => log(err) *> return an empty list of modif for that part)`

Reply:
ha

Reply:
oh it's outdated :)

Reply:
i found a solution when i pressed enter :)
i need to check it works